Fix error seen with clang 8:
error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
https://bugs.launchpad.net/pbzip2/+bug/1746369
--- pbzip2.cpp.orig
+++ pbzip2.cpp
@@ -1220,7 +1220,7 @@ int producer_decompress(int hInfile, OFF_T fileSize, queue *fifo)
 			// give warning to user if block is larger than 250 million bytes
 			if (fileData->bufSize > 250000000)
 			{
-				fprintf(stderr, "pbzip2:  *WARNING: Compressed block size is large [%"PRIuMAX" bytes].\n",
+				fprintf(stderr, "pbzip2:  *WARNING: Compressed block size is large [%" PRIuMAX " bytes].\n",
 						(uintmax_t) fileData->bufSize);
 				fprintf(stderr, "          If program aborts, use regular BZIP2 to decompress.\n");
 			}
@@ -1829,7 +1829,7 @@ void *fileWriter(void *outname)
 	
 	if (QuietMode != 1)
 	{
-		fprintf(stderr, "    Output Size: %"PRIuMAX" bytes\n", (uintmax_t)CompressedSize);
+		fprintf(stderr, "    Output Size: %" PRIuMAX " bytes\n", (uintmax_t)CompressedSize);
 	}
 
 	#ifdef PBZIP_DEBUG
@@ -2072,7 +2072,7 @@ int directcompress(int hInfile, OFF_T fileSize, int blockSize, const char *OutFi
 		close(hOutfile);
 	if (QuietMode != 1)
 	{
-		fprintf(stderr, "    Output Size: %"PRIuMAX" bytes\n", (uintmax_t)CompressedSize);
+		fprintf(stderr, "    Output Size: %" PRIuMAX " bytes\n", (uintmax_t)CompressedSize);
 	}
 
 	syncSetProducerDone(1); // Not really needed for direct version
@@ -2794,7 +2794,7 @@ void outputBufferInit(size_t size)
 	// unlikely to get here since more likely exception will be thrown
 	if (OutputBuffer.size() != size)
 	{
-		fprintf(stderr, "pbzip2: *ERROR: Could not initialize (OutputBuffer); size=%"PRIuMAX"!  Aborting...\n", (uintmax_t)size);
+		fprintf(stderr, "pbzip2: *ERROR: Could not initialize (OutputBuffer); size=%" PRIuMAX "!  Aborting...\n", (uintmax_t)size);
 		safe_mutex_unlock(OutMutex);
 		exit(1);
 	}
@@ -4102,7 +4102,7 @@ int main(int argc, char* argv[])
 			if (decompress == 1)
 				fprintf(stderr, " BWT Block Size: %c00k\n", BWTblockSizeChar);
 			if (hasInFile)
-				fprintf(stderr, "     Input Size: %"PRIuMAX" bytes\n", (uintmax_t)InFileSize);
+				fprintf(stderr, "     Input Size: %" PRIuMAX " bytes\n", (uintmax_t)InFileSize);
 		}
 
 		if (decompress == 1)
@@ -4224,7 +4224,7 @@ int main(int argc, char* argv[])
 			{
 				NumBufferedBlocksMax = numCPU;
 				if (QuietMode != 1)
-					fprintf(stderr, "*Warning* Max memory limit increased to %"PRIuMAX" MB to support %d CPUs\n", (uintmax_t)((NumBufferedBlocksMax + (numCPU * 2)) * blockSize)/1000000, numCPU);
+					fprintf(stderr, "*Warning* Max memory limit increased to %" PRIuMAX " MB to support %d CPUs\n", (uintmax_t)((NumBufferedBlocksMax + (numCPU * 2)) * blockSize)/1000000, numCPU);
 			}
 			else
 			{
